sway-core/src/language/ty/expression/contract.rs

```diff
@@ -2,7 +2,7 @@ use crate::language::ty::*;
 
 #[derive(Clone, Debug)]
 pub struct ContractCallParams {
-    pub(crate) func_selector: [u8; 4],
+    pub(crate) func_selector: Option<[u8; 4]>,
```

Comment:
I think this needs a comment to explain what the type means for the `None` case.

Reply:
Done.